### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by dan-lee**
Update CONTRIBUTING template

<s>This exposes the Date object to not rely on the exposed stringified (and localized?) month.
For example, this makes checking if the current month is active easier.
This is according to the exposed `$date` object on a single day.</s>

edit: Can be achieved like so:
```ts
const months = useMonths(dpState)
const activeMonth = Object.values(months)[0].find((month) => month.active)
```

### CODE REVIEW

**Review by Feshchenko — COMMENTED** [review]
Hi, nice work.
Left one comment in the code.
General comments:
- please update version in the package.json to `5.2.1`
- update tests which rely on the `createCalendar` util.

File: src/utils/create-calendars.ts [hunk 1]
```diff
@@ -59,6 +59,7 @@ const createCalendar = (
   }
 
   return {
+    $date: offsetDate,
```
Comment (Feshchenko) [inline src/utils/create-calendars.ts:62]: - what is the use case for this? months has `active` prop for usecase from the PR description.

Comment (dan-lee) [reply src/utils/create-calendars.ts:62]: Aha! I didn't find this information before, seems like this does also work:

```ts
const months = useMonths(dpState)
const activeMonth = Object.values(months)[0].find((month) => month.active)
```

This PR might offer some minor convenience then.
@Feshchenko you can decide if you still want that in then :)

Comment (Feshchenko) [reply src/utils/create-calendars.ts:62]: Yea, let's keep this PR.
You should make this steps:
- rebase with master to get green tests.
- delete `$date` from calendars.
- increase version in the package.json

**Review by Feshchenko — APPROVED** [review]

**Pull request merged**